packages/verser/src/lib/verser-connection.ts

```diff
@@ -149,4 +149,15 @@ export class VerserConnection {
             // TODO: Error handling?
         });
     }
+
+    /**
+     * Closes the connection by sending FIN packet.
+     *
+     * @returns Promise resolving when connection is ended.
+     */
+    async close() {
+        return new Promise<void>(res => {
+            this.socket.end(res);
```

Comment:
Maybe to add somewhere in close() method the this.logger.log(..) ?

Reply:
Added in https://github.com/scramjetorg/transform-hub/pull/196/commits/25c72f0bbb183d103020e8e6ecb864a97d688998 :+1: